kobo/apps/trackers/tests.py

```diff
@@ -0,0 +1,94 @@
+import datetime
```

Comment:
You can import `from datetime import datetime` to avoid calling `datetime.datetime` everywhere.